nitransforms/io/lta.py

```diff
@@ -92,7 +92,7 @@ def from_string(klass, string):
 
 class LinearTransform(StringBasedStruct):
     template_dtype = np.dtype([
-        ('mean', 'f4', (3, 1)),       # x0, y0, z0
+        ('mean', 'f4', (3, 1)),  # x0, y0, z0
         ('sigma', 'f4'),
         ('m_L', 'f4', (4, 4)),
```

Comment:
```suggestion
        ('m_L', 'f8', (4, 4)),
```
I believe that using double precision will make the conversion test more accurate. We should migrate all affines to double precision.

Reply:
ha, this was the first thing I thought of and tried with the earlier files, but there was still a large difference. will give it a try with these new transforms

Reply:
we should use double precision for all affines, anyways.